### PR TITLE
Update nl.js changes 2/2/2022

### PR DESCRIPTION
**Pull request by nickauthentic**
Added some new translations :)

### CODE REVIEW

**Review by gluzly — APPROVED** [review]
Perfect!

File: nl.js [hunk 1]
```diff
@@ -91,15 +91,15 @@ settings.network.header Netwerk
 settings.network.netRerouting Auto Routing (Beïnvloed Ping)
 settings.network.netRate Netwerk snelheid (Hz)
 settings.network.lagComp Lag Compensatie
-settings.network.optNet Optimized Networking
+settings.network.optNet Geoptimaliseerde Netwerken
```
Comment (AdoreMe-GitHub) [inline nl.js:94]: Correct

File: nl.js [hunk 2]
```diff
@@ -91,15 +91,15 @@ settings.network.header Netwerk
...
 settings.popup.header Popups
 settings.experimental.header Experimenteel
-settings.experimental.holiday Holiday Effects
+settings.experimental.holiday Holiday Effecten
```
Comment (AdoreMe-GitHub) [inline nl.js:97]: I would change it to "Feestlijke Effecten"

**Review by AdoreMe-GitHub — CHANGES_REQUESTED** [review]
97. settings.experimental.holiday Holiday Effecten needs to be settings.experimental.holiday Feestlijke effecten

101. settings.experimental.mouseAccel Muis Versnelling needs to be settings.experimental.mouseAccel Muis Acceleratie

221. settings.medals.show Show Medailles needs to be settings.medals.show Weergeef Medailles

548. windows.controls.propRand Selecteer Willekeurig Prop needs to be windows.controls.propRand Selecteer Willekeurige Prop

**Review by AdoreMe-GitHub — APPROVED** [review]
Correctly corrected it 👍

**Comment by mty22** [comment]
Hi everyone, thanks for the PR and the efficiency around it.

It's also good to see the original translators back.

Can you all confirm your IGNs, please?

**Comment by nickauthentic** [comment]
> here you go frzn

https://krunker.io/social.html?p=profile&q=NickAuthentic
https://krunker.io/social.html?p=profile&q=AdoreMe
https://krunker.io/social.html?p=profile&q=Gluzly

**Pull request merged**